### PR TITLE
CDAP-7803 update market docs

### PR DESCRIPTION
**Pull request by albertshau**
Update the market docs to reflect what the UI currently has.

### CODE REVIEW

File: cdap-docs/integrations/source/cask-market.rst [hunk 1]
```diff
@@ -310,9 +316,9 @@ Example action::
     ]
   }
 
-create_artifact
+create_driver_artifact
 ...............
```
Comment (awholegunch) [inline cdap-docs/integrations/source/cask-market.rst]: add more periods to match length of line above

File: cdap-docs/integrations/source/cask-market.rst [hunk 2]
```diff
@@ -392,6 +394,78 @@ where ``mysql-connector-java.json`` is a package resource with content such as::
     "properties": { }
   }
 
+create_plugin_artifact
+...............
```
Comment (awholegunch) [inline cdap-docs/integrations/source/cask-market.rst]: add more periods to match length of line above

File: cdap-docs/integrations/source/cask-market.rst [hunk 3]
```diff
@@ -500,30 +574,11 @@ Example action::
...
-     - Application name
-     - Yes
-     -
-   * - ``config``
-     - Package resource containing the application config
-     - No
-     -
+Deploys a CDAP application by prompting the user to upload the application jar.
+Does not take any arguments. In the future, you will be able to specify the jar
+as a package resource.
```
Comment (awholegunch) [inline cdap-docs/integrations/source/cask-market.rst]:
```
Deploys a CDAP application by prompting the user to upload the application JAR.
Does not take any arguments. In a future release, you will be able to specify
the JAR as a package resource.

File: cdap-docs/integrations/source/cask-market.rst [hunk 4]
```diff
@@ -310,9 +316,9 @@ Example action::
     ]
   }
 
-create_artifact
+create_driver_artifact
 ...............
-Creates a CDAP artifact.
+Creates a CDAP artifact containing a third party JDBC Driver.
```
Comment (awholegunch) [inline cdap-docs/integrations/source/cask-market.rst]:
```
Creates a CDAP artifact containing a third-party JDBC Driver.

File: cdap-docs/integrations/source/cask-market.rst [hunk 5]
```diff
@@ -120,6 +120,7 @@ This will return a JSON array that lists each package and its metadata:
         "org": "Cask Data, Inc.",
         "cdapVersion": "|release-range|",
         "created": 1473901763,
+        "beta": false,
```
Comment (awholegunch) [inline cdap-docs/integrations/source/cask-market.rst:123]: does this one need a "license.txt"?

**Comment by awholegunch** [comment]
Minor nits, otherwise LGTM. Please fix and merge!

File: cdap-docs/integrations/source/cask-market.rst (hunk 5 above)
Comment (albertshau) [reply cdap-docs/integrations/source/cask-market.rst:123]: nope

**Pull request merged**